docs/developer/guidelines/dashboards.md

```diff
@@ -67,7 +67,7 @@ Follow the guidelines below as you build your dashboard to achieve something sim
 1. Add a logo to the dashboard header. The integration logo will automatically appear in the header if the icon exists [here](http://app.datadoghq.com/internal/styleguide/logos/logo-gallery?variant=avatar) and the `integration_id` matches the icon name. That means it will only appear when the dashboard you're working on is made into the official integration board.
    ![Logo in header](../assets/images/logo_header.png)
 
-1. Always include an "about" group for the integration containing a brief description and helpful links. Also include an "overview" group containing a few of the most important metrics, and place it at the top of the dashboard. Edit the "about" group and select the "banner" display option, then link to a banner image like this: `/static/images/integration_dashboard/your-image.png`. For instructions on how to create and upload a banner image, go to the [DRUIDS logo gallery](http://app.datadoghq.com/internal/styleguide/logos/logo-gallery), click the relevant logo, and click the "MSL Dashboard" tab. The "about" section should contain content, not data; the "overview" section should contain data. Avoid making the "about" section full-width.
+1. Always include an "about" group for the integration containing a brief description and helpful links. Also include an "overview" group containing a few of the most important metrics, and place it at the top of the dashboard. Edit the "about" group and select the "banner" display option, then link to a banner image like this: `/static/images/integration_dashboard/your-image.png`. For instructions on how to create and upload a banner image, go to the [DRUIDS logo gallery](https://druids.us1.prod.dog/logos/logo-gallery), click the relevant logo, and click the "Dashboard Banner" tab. The "about" section should contain content, not data; the "overview" section should contain data. Avoid making the "about" section full-width.
```

Comment:
I think the "official" address is https://druids.datadoghq.com/logos/logo-gallery

Reply:
```suggestion
1. Always include an "about" group for the integration containing a brief description and helpful links. Also include an "overview" group containing a few of the most important metrics, and place it at the top of the dashboard. Edit the "about" group and select the "banner" display option, then link to a banner image like this: `/static/images/integration_dashboard/your-image.png`. For instructions on how to create and upload a banner image, go to the [DRUIDS logo gallery](https://druids.datadoghq.com/logos/logo-gallery), click the relevant logo, and click the "Dashboard Banner" tab. The "about" section should contain content, not data; the "overview" section should contain data. Avoid making the "about" section full-width.
```